### DISCUSSION TITLE
Vercel builds fail, error continues to point to file that I already removed

### DISCUSSION THREAD

**Discussion by richleach** [vercel/vercel#8938] [Help] [answered]
Hello and thank you for helping me. 

I'm executing a deployment for my Next.js site and I'm troubleshooting error messages. The first error is
 
"warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json."

So, as per the error message I went and removed package-lock.json from my git repo. Unfortunately, and even though the file no longer exists, I continue to get the error message (above).
I double checked and I am NOT redeploying with existing build cache (that checkbox is unchecked). It seems like some sort of caching is continuing to happen, does anyone know how I "jiggle the handle"?
Thanks in advance,
Rich

**Comment by amyegan** [comment, marked as answer]
This question seems to be the same as https://github.com/vercel/community/discussions/1066. Let's keep the conversation all in one place over there! 😄
